### ISSUE TITLE
[suggestion] Proposal to OCA

### ISSUE THREAD

**Issue by alejandrosantana** [OpenSur/Odoo_addons#20]
Hi. Have you considered proposing _restrict_db_mgr_ to as an OCA module in https://github.com/OCA/web repo?

By the way, it actually does not hide the link.
I managed to hide it in _web_hide_db_manager_link_ (see https://github.com/OCA/web/pull/89 )

**Comment by lukebranch** [comment]
+1

I really recommend it. These modules are very useful!

**Comment by OpenSur** [comment]
Yes, we didn't hide the link, we focused in the hard part avoiding people who knows how to access directly. I think put it in OCA is a good idea, which is the procedure for that?

**Comment by alejandrosantana** [comment]
The procedure usually starts with signing a CLA (http://odoo-community.org/page/cla).
I am not really sure if you need to become a paid member (50€/year, not much), but you can ask any doubt at contribute@odoo-community.org

Regarding the tiny bit of hiding the link, see the pull request I tell you about, it's done there, but your module is much more interesting, I think.
